src/tests/Equinor.ProCoSys.IPO.WebApi.Tests/Synchronization/BusReceiverServiceTests.cs

```diff
@@ -174,45 +189,10 @@ public async Task HandlingMcPkgTopicWithoutFailure()
             _unitOfWork.Verify(u => u.SaveChangesAsync(It.IsAny<CancellationToken>()), Times.Once);
             _plantSetter.Verify(p => p.SetPlant(plant), Times.Once);
             _invitationRepository.Verify(i => i.UpdateMcPkgOnInvitations(project1Name, mcPkgNo1, description), Times.Once);
-            _invitationRepository.Verify(i => i.UpdateCommPkgOnInvitations(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>()), Times.Never);
-            _invitationRepository.Verify(i => i.UpdateProjectOnInvitations(It.IsAny<string>(), It.IsAny<string>()), Times.Never);
-        }
-
-        [TestMethod]
-        public async Task HandlingMcPkgTopicWithoutFailure_WhenMoveMcPkg()

```

Comment:
As mentioned in BusReceiverService.cs when handling a mcpkgtopic, I don't see how we now handle mcpkg move ... If we do, an unit test should tell me so